Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.3-onbuild
+
+EXPOSE 9102
+EXPOSE 9125/udp
+
+VOLUME ["/mapping.conf"]
```

Comment:
I would omit this since it would force people to use a volume where I would rather create a custom image based on this one. You can still use a bind-mount even without this volume definition.